src/test/java/org/opentripplanner/updater/vehicle_positions/VehiclePositionParsingTest.java

```diff
@@ -0,0 +1,40 @@
+package org.opentripplanner.updater.vehicle_positions;
+
+import java.net.URI;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class VehiclePositionParsingTest {
+
+    @Test
```

Comment:
Can these two tests become a `@ParameterizedTest`?

Reply:
Not really, because while they may look identical they test slightly different scenarios.